### PR TITLE
Ja feature map view

### PR DESCRIPTION
**Pull request by jeremiahaguirre**
- Add pin functionality
- Locator and add pin buttons refactored

### CODE REVIEW

File: client/js/screens/FindLocker/styles.js [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+//     top: "50%",
+//     textAlign: "center"
+//   },
+//   map: {
+//     height: "100%"
+//   }
+// });
 
-export default styles;
+// export default styles;
```
Comment (teelau) [inline client/js/screens/FindLocker/styles.js]: Remove commented stuff

File: client/js/components/MapView/MapView.js [hunk 2]
```diff
@@ -54,61 +63,113 @@ class MapViewComponent extends Component {
...
-                <TouchableOpacity
-                  style={styles.locator}
-                  onPress={() => this.getCurrentLocation()}
+              <View style={styles.mapContainer}>
+                <MapView
+                  provider={PROVIDER_GOOGLE}
+                  style={styles.mapView}
+                  onLongPress={e => {
+                    console.log(e);
+                    console.log(e.target);
```
Comment (teelau) [inline client/js/components/MapView/MapView.js]: Remove console logs

File: client/js/screens/FindLocker/FindLocker.js [hunk 3]
```diff
@@ -9,8 +9,8 @@ const FindLocker = () => {
   return (
     <GooglePlaceContext.Consumer>
       {({ location, updateLocation }) => (
-        <View style={styles.map}>
-          <GoogleMapsSearch updateContextLocation={updateLocation} />
+        <View>
+          {/* <GoogleMapsSearch updateContextLocation={updateLocation} /> */}
```
Comment (teelau) [inline client/js/screens/FindLocker/FindLocker.js]: Remove comment

File: client/ios/Podfile.lock [hunk 4]
```diff
@@ -155,4 +155,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 651aa25d6d26c9ddbf82fb371f80902b69d52210
 
-COCOAPODS: 1.7.2
+COCOAPODS: 1.7.1
```
Comment (siddharthparmar7) [inline client/ios/Podfile.lock:158]: did you version down?

File: client/js/components/MapView/MapView.js [hunk 5]
```diff
@@ -1,20 +1,28 @@
...
       latitude: 49.2827,
       longitude: 123.1207,
-      error: null
+      coordinates: {
+        latitude: null,
+        longitude: null
+      },
+      slider: false,
+      error: null,
+      x: null
```
Comment (siddharthparmar7) [inline client/js/components/MapView/MapView.js]: what is x?

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes.

File: client/ios/Podfile.lock (hunk 4 above)
Comment (jeremiahaguirre) [reply client/ios/Podfile.lock:158]: No I didn't

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**